src/tink/json/macros/Macro.hx

```diff
@@ -91,6 +91,9 @@ class Macro {
   }
   
   static public function getRepresentation(t:Type, pos:Position) {
+    
+    if(t.match(TMono(_))) return None;
```

Comment:
This is to guard infinite loop if something happens to be `Unknown<0>`.
For example, it happens when doing `tink.Json.stringify(None)`, where the type is `Option<Unknown>`